docs/minishift_start.md

```diff
@@ -14,42 +14,41 @@ minishift start
-      --insecure-registry value    Non-secure Docker registries to pass to the Docker daemon. (default [172.30.0.0/16])
-      --iso-url string             Location of the minishift ISO. (default "https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.0/minishift-b2d.iso")
-      --memory int                 Amount of RAM to allocate to the Minishift VM. (default 2048)
-      --metrics                    Install metrics (experimental)
-  -e, --openshift-env value        Specify key-value pairs of environment variables to set on the OpenShift container. (default [])
-      --openshift-version string   The OpenShift version to run, eg. v1.4.1 (default "v1.4.1")
-      --public-hostname string     Public hostname of the OpenShift cluster.
-      --registry-mirror value      Registry mirrors to pass to the Docker daemon. (default [])
-      --routing-suffix string      Default suffix for the server routes.
-      --server-loglevel int        Log level for the OpenShift server.
-      --skip-registry-check        Skip the Docker daemon registry check.
-      --vm-driver string           The driver to use for the Minishift VM. Possible values: [virtualbox vmwarefusion kvm xhyve hyperv] (default "kvm")
+      --cpus int                        Number of CPU cores to allocate to the Minishift VM. (default 2)
+      --disk-size string                Disk size to allocate to the Minishift VM. Use the format <size><unit>, where unit = b, k, m or g. (default "20g")
+      --docker-env stringArray          Environment variables to pass to the Docker daemon. Use the format <key>=<value>.
+      --forward-ports                   Use Docker port forwarding to communicate with the origin container. Requires 'socat' locally.
+      --host-config-dir string          Location of the OpenShift configuration on the Docker host. (default "/var/lib/minishift/openshift.local.config")
+      --host-data-dir string            Location of the OpenShift data on the Docker host. If not specified, etcd data will not be persisted on the host. (default "/var/lib/minishift/hostdata")
+      --host-only-cidr string           The CIDR to be used for the minishift VM. (Only supported with VirtualBox driver.) (default "192.168.99.1/24")
+      --host-volumes-dir string         Location of the OpenShift volumes on the Docker host. (default "/var/lib/origin/openshift.local.volumes")
+      --insecure-registry stringSlice   Non-secure Docker registries to pass to the Docker daemon. (default [172.30.0.0/16])
+      --iso-url string                  Location of the minishift ISO. (default "https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.0/minishift-b2d.iso")
+      --memory int                      Amount of RAM to allocate to the Minishift VM. (default 2048)
+      --metrics                         Install metrics (experimental)
+  -e, --openshift-env stringSlice       Specify key-value pairs of environment variables to set on the OpenShift container.
```

Comment:
We should investigate. IMO this is a bug in Viper then. If there is nothing in Viper, we can always control the generation and fix afterwards, but that's not something worth doing right now.